thingsvision/core/extraction/helpers.py

```diff
@@ -129,17 +134,34 @@ def create_model_extractor(
     backend: str
         The backend of the model. Either "pt" for PyTorch or "tf" for TensorFlow.
     forward_fn: Callable
-        In case your model requires more complicated forward passes than simply using model(img),
-        you can pass a custom forward function here. The function must have the following signature:
-
-        forward_fn(self, img, module_name) -> activations
-
-        and calls to the model have to be made on the self.model attribute.
+        In case your model requires more complicated forward passes than the default,
+        for example if the forward pass requires multiple inputs, you can pass a custom
+        forward function here, which should take a batch of images as input. Only required
+        for PyTorch models.
+
+        Default:
+            def forward(
+                self, batch: TensorType["b", "c", "h", "w"]
+            ) -> TensorType["b", "num_cls"]:
+                return self.model(batch)
+
+    flatten_fn: Callable
```

Comment:
I think `flatten_fn` (and potentially `forward_fn` as well) need to be explained in the docs. Since this is a new addition, there's no clear pointer yet as to when or why a user wants to use that.